src/Process/Listener/MutationTestingConsoleLoggerSubscriber.php

```diff
@@ -95,18 +95,29 @@ public function onMutationTestingFinished(MutationTestingFinished $event)
     {
         // TODO [doc] write test -> run mutation for just this file. Should be 100%, 100%, 100%,
         $this->outputFormatter->finish();
-        $processes = $this->metricsCalculator->getEscapedMutantProcesses();
 
         if ($this->showMutations) {
-            $this->showMutations($processes);
+            $this->showMutations(
+                $this->metricsCalculator->getEscapedMutantProcesses(),
+                'Escaped'
+            );
+
+            $this->showMutations(
```

Comment:
I'm -1 on this addition, because from my experiense almost every big enough project has thousands (literally) uncovered mutations. It will just pollute the terminal without possibility to scroll up (because of terminal limitations).

Such long text should be inside text file only - `infection-log.txt`.

Reply:
I thought about it, too. We could adjust the detail of the output relying on the "-vvv" option.

Reply:
Yes, let's have displayed escaped mutations by default with `-s`, and uncovered with a verbose level

Reply:
Ok. I will fix this PR as soon as possible.